.github/workflows/python-package.yml

```diff
@@ -119,3 +121,42 @@ jobs:
               run: >
                   tox --verbose --verbose -e
                   "${{ matrix.python-version }}-integration-${{ matrix.toxenv }}" -vv
+
+  Smoke:
+      needs:
+      - Integration
+      if: needs.Integration.result == 'success'
+      timeout-minutes: 240
+
+      runs-on: ubuntu-latest
+      strategy:
+          fail-fast: false
+          matrix:
+              python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+
+      steps:
+          -   name: Fetch Docker Images
+              run: |
+                docker pull redis:latest
+                docker pull rabbitmq:latest
+
+          -   name: Install apt packages
+              run: |
+                  sudo apt update
+
+          -   uses: actions/checkout@v3
+          -   name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v4
```

Comment:
```suggestion
          -   uses: actions/checkout@v4
          -   name: Set up Python ${{ matrix.python-version }}
              uses: actions/setup-python@v5
```

Reply:
Nice catch - this is automatically updated via bots though.
That being said, I’m preparing a hotfix to fix it thanks to you :)

This branch is built off multiple standard PRs and will be merged as a list of commits, so I do not want to commit changes without a proper PR.